### PR TITLE
<Accordion/> - introduce builders

### PR DESCRIPTION
**Pull request by EgoziE**
Builders for Accordion! (with full backwards compatibility)
image

### CODE REVIEW

**Comment by EgoziE** [comment]
```
<Accordion
  items={[
    accordionSectionItemBuilder({
      title: "I'm an Accordion Section Item !",
    }),
    accordionItemBuilder({
      title: 'First Row',
      children: (
        <Text>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
          eiusmod tempor incididunt ut labore et dolore magna aliqua
        </Text>
      ),
    }),
  ]}
/>
```

**Comment by EgoziE** [comment]
@shlomitc This is needed because we got request for a "section accordion item"
I'd love to hear your thoughts about this PR

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6442

**Review by TalZa1810 — APPROVED** [review]
Seems fine.
Just for transparency- 
this is PR is to help our contributors with the `AccordionItemSection` contribution.

**Comment by TalZa1810** [comment]
Another thought- maybe adding a unit test for the render function should be added. Something similar to [this](https://github.com/wix/wix-style-react/blob/master/src/DropdownLayout/test/DropdownLayout.spec.js#L282).

**Review by TalZa1810 — CHANGES_REQUESTED** [review]
Another thought- maybe adding a unit test for the render function should be added. Something similar to [this](https://github.com/wix/wix-style-react/blob/master/src/DropdownLayout/test/DropdownLayout.spec.js#L282).

File: src/Accordion/Accordion.js [hunk 1]
```diff
@@ -22,22 +22,25 @@ class Accordion extends React.Component {
...
-        disabled: PropTypes.bool,
-        onToggle: PropTypes.func,
-        onMouseEnter: PropTypes.func,
-        onMouseleave: PropTypes.func,
-        open: PropTypes.bool,
-        initiallyOpen: PropTypes.bool,
-      }),
-    ),
+    items: PropTypes.oneOfType([
+      PropTypes.arrayOf(PropTypes.func),
```
Comment (shlomitc) [inline src/Accordion/Accordion.js:26]: is this still correct?. I see that you now expect a `render()` method on the items object

File: src/Accordion/test/Accordion.spec.js [hunk 2]
```diff
@@ -235,4 +236,77 @@ describe('Accordion', () => {
...
+      expect(await driver.isItemExpandedAt(1)).toBe(false);
+
+      await driver.clickToggleButtonAt(0);
+
+      expect(await driver.isItemExpandedAt(0)).toBe(true);
+      expect(await driver.isItemExpandedAt(1)).toBe(false);
+
+      await driver.clickToggleButtonAt(1);
+
+      await sleep(350).then(async () => {
```
Comment (shlomitc) [inline src/Accordion/test/Accordion.spec.js:306]: doesn't it make sense that the driver will do the `sleep`ing?

Comment (EgoziE) [reply src/Accordion/test/Accordion.spec.js:306]: Good idea! thanks

**Pull request merged**